spec/forms/moves/information_spec.rb

```diff
@@ -98,6 +98,20 @@
           expect(subject.errors).to include :date
         end
       end
+
+      context "with a date in the past" do
+        it 'returns false' do
+          params[:date] = '01/01/2015'
+          expect(subject.validate(params)).to be false
+        end
+
+        it 'sets a descriptive error on date' do
+          params[:date] = '01/01/2015'
+          subject.validate(params)
+          expect(subject.errors).to include :date
```

Comment:
Might not need this check as the one beneath validates that `:date` is present in order to return the error msg.